web_external/js/views/widgets/StyleWmsDatasetWidget.js

```diff
@@ -133,8 +136,15 @@ minerva.views.StyleWmsDatasetWidget = minerva.View.extend({
                 max: maximum,
                 ramps: this.ramps
             });
+        } else if (subType === 'point') {
```

Comment:
Where does this point subtype get set?  In the ingestion, I only see single and multiband subtypes.